We satisfy the ninja dependency directly by the developer/build/ninja package
and not via the ninja-python-distributions (https://pypi.org/project/ninja).

--- dbus-python-1.3.2/pyproject.toml.orig
+++ dbus-python-1.3.2/pyproject.toml
@@ -6,7 +6,6 @@
 requires = [
     'meson-python>=0.8.1',
     'meson>=0.60.0',
-    'ninja',
     'patchelf',
     'setuptools',
     'wheel',
